pkg/certificates/mocks/mock_client.go

```diff
@@ -0,0 +1,58 @@
+// Code generated by MockGen. DO NOT EDIT.
+// Source: storj.io/storj/pkg/pb (interfaces: CertificatesClient)
+
+// Package mocks is a generated GoMock package.
+package mocks
+
+import (
+	context "context"
+	reflect "reflect"
+
+	gomock "github.com/golang/mock/gomock"
+	grpc "google.golang.org/grpc"
+
+	pb "storj.io/storj/pkg/pb"
+)
+
+// MockCertificatesClient is a mock of CertificatesClient interface
+type MockCertificatesClient struct {
```

Comment:
Is there a way this mock can be avoided and use an actual implementation instead? We have missed a bunch of bugs due to using mocks.

Reply:
This mock is only used for testing the client. The server method has a test as well as an end to end test using an actual client and running grpc server.

Reply:
Sure, I just want to avoid adding new mock code. I like to see most of the mocks disappear from the code-base. So yeah, we can use the actual server for testing rather than the mock. For the error conditions we can use a erroring wrapper if necessary.